examples/seismic/viscoacoustic/operators.py

```diff
@@ -0,0 +1,208 @@
+import sympy as sp
+import numpy as np
+
+from devito import (Eq, Operator, VectorTimeFunction, TimeFunction, NODE,
+                    div, grad, diag)
+from examples.seismic import PointSource, Receiver
+
+def viscoacoustic_blanch_symes(v, r, p, vp, irho, rho, t_s, tt, s, damp, **kwargs):
```

Comment:
You should be able to call it as `forward(model, r, p, t_s, tt)` has all the physical parameters are in the model.